commands/xdel.md

```diff
@@ -7,7 +7,7 @@ however Redis streams are represented in memory, so we are also able to
 delete entries. This may be useful, for instance, in order to comply with
 certain privacy policies.
 
-# Understanding the low level details of entries deletion
+## Understanding the low level details of entries deletion
```

Comment:
i like zadd style, like https://redis.io/commands/zadd/#zadd-options.
but `#` is definitely wrong, thanks, let's wait for the opinion of others
https://redis.io/commands/xdel/#understanding-the-low-level-details-of-entries-deletion

<img width="863" alt="image" src="https://user-images.githubusercontent.com/22811481/166098343-76b8b582-b323-4ae2-ac8b-2710b9adba31.png">


```suggestion
Understanding the low level details of entries deletion
---
```